tests/factories.py

```diff
@@ -243,32 +243,17 @@ class Meta:
 class ConferenceFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = "conference.Conference"
+        django_get_or_create = ('name',)
```

Comment:
doesn't make much difference but I'd go with get_or_create on code instead, because it's specified in settings and iirc conference code appears in a couple of places in db w/o a FK to Conference.

Reply:
That's a good idea - even more so as `code` is the PK